x-pack/legacy/plugins/index_management/public/application/index.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render, unmountComponentAtNode } from 'react-dom';
+
+import { CoreStart } from '../../../../../../src/core/public';
+
+import { AppContextProvider, AppDependencies } from './app_context';
+import { App } from './app';
+import { indexManagementStore } from './store';
+
+export const renderApp = (
+  elem: HTMLElement | null,
+  { core, dependencies }: { core: CoreStart; dependencies: AppDependencies }
+) => {
+  if (!elem) {
+    return () => undefined;
+  }
+
+  const { i18n } = core;
+  const { Context: I18nContext } = i18n;
+  const { services } = dependencies;
+
+  render(
```

Comment:
Does this application use a router? If so it needs to be configured with `appBasePath` to ensure it works with the basePath correctly. Once #56705 is merged, you will need to use the `history` instance that is provided with your react-router.

Reply:
Setting the `basename` on the react router does not work as we already add the base path manually on each route like so

```js
<Switch>
  <Route exact path={`${BASE_PATH}create_template`} component={TemplateCreate} />
  <Route exact path={`${BASE_PATH}clone_template/:name*`} component={TemplateClone} />
  <Route exact path={`${BASE_PATH}edit_template/:name*`} component={TemplateEdit} />
  <Route path={`${BASE_PATH}:section(indices|templates)`} component={IndexManagementHome} />
  <Redirect from={`${BASE_PATH}`} to={`${BASE_PATH}indices`} />
</Switch>
```

Do you recommend to refactor and use the `basename` prop on the router?